portal/server/utils/csv-utils.js

```diff
@@ -13,6 +13,17 @@ const columnIndexToExcelColumn = (index) => {
   return result;
 };
 
+/**
+ * Extracts the value in the cell of an excel cell and removes any new lines so that it doesn't affect parsing as a csv.
+ * @param {Object} cell - excel cell.
+ * @returns {string | number} - cell value.
+ */
+const extractCellValue = (cell) => {
+  const cellValue = cell.value?.result ? cell.value.result : cell.value;
+  const cellValueWithoutNewLines = typeof cellValue === 'string' ? cellValue?.replace(/\n/g, ' ') : cellValue;
```

Comment:
Could use `/(\r\n|\r|\n)+/g` for the regex here to account for other possible new line characters

Reply:
Yeah I thought about this although the library only cares about `\n` as the condition for a new line in csv. Maybe it's helpful for me to replace \r\n with a space instead thought as otherwise it'll be '\r ' which is probably fine but weird!